src/test/scala/org/hammerlab/guacamole/commands/GermlineAssemblyCallerSuite.scala

```diff
@@ -21,188 +21,138 @@ class GermlineAssemblyCallerSuite extends FunSuite with Matchers with BeforeAndA
-    variant1.start should be(1297212)
-    Bases.basesToString(variant1.allele.refBases) should be("G")
-    Bases.basesToString(variant1.allele.altBases) should be("C")
-
-    val variant2 = variants(1)
-    variant2.referenceContig should be("chr1")
-    variant2.start should be(1297215)
-    Bases.basesToString(variant2.allele.refBases) should be("A")
-    Bases.basesToString(variant2.allele.altBases) should be("G")
-
-  }
-
-  test("test assembly caller: illumina platinum tests; 2 nearby homozygous snps") {
-
-    val loci = "chr1:1316669-1316670"
-    val variants = discoverGenotypesAtLoci(loci)
-
-    variants.length should be(3)
-    val variant1 = variants(0)
-    variant1.referenceContig should be("chr1")
-    variant1.start should be(1316647)
-    Bases.basesToString(variant1.allele.refBases) should be("C")
-    Bases.basesToString(variant1.allele.altBases) should be("T")
-
-    val variant2 = variants(1)
-    variant2.referenceContig should be("chr1")
-    variant2.start should be(1316669)
-    Bases.basesToString(variant2.allele.refBases) should be("C")
-    Bases.basesToString(variant2.allele.altBases) should be("G")
-
-    val variant3 = variants(2)
-    variant3.referenceContig should be("chr1")
-    variant3.start should be(1316673)
-    Bases.basesToString(variant3.allele.refBases) should be("C")
-    Bases.basesToString(variant3.allele.altBases) should be("T")
-
-  }
-
-  test("test assembly caller: illumina platinum tests; het snp") {
-
-    val loci = "chr1:1342611-1342612"
-    val variants = discoverGenotypesAtLoci(loci)
 
-    variants.length should be(1)
-    val variant = variants(0)
-    variant.referenceContig should be("chr1")
-    variant.start should be(1342611)
-    Bases.basesToString(variant.allele.refBases) should be("G")
-    Bases.basesToString(variant.allele.altBases) should be("C")
-  }
-
-  test("test assembly caller: illumina platinum tests; homozygous deletion") {
-
-    val loci = "chr1:1296368-1296369"
-    val variants = discoverGenotypesAtLoci(loci)
-
-    variants.length should be(1)
-    val variant = variants(0)
-    variant.referenceContig should be("chr1")
-    variant.start should be(1296368)
-    Bases.basesToString(variant.allele.refBases) should be("GAC")
-    Bases.basesToString(variant.allele.altBases) should be("G")
-  }
-
-  test("test assembly caller: illumina platinum tests; homozygous deletion 2") {
-
-    val loci = "chr1:1303426-1303427"
-    val variants = discoverGenotypesAtLoci(loci)
-
-    variants.length should be(1)
-    val variant = variants(0)
-    variant.referenceContig should be("chr1")
-    variant.start should be(1303426)
-    Bases.basesToString(variant.allele.refBases) should be("ACT")
-    Bases.basesToString(variant.allele.altBases) should be("A")
-  }
-
-  test("test assembly caller: illumina platinum tests; homozygous insertion") {
-
-    val loci = "chr1:1321298-1321299"
-    val variants = discoverGenotypesAtLoci(loci)
+  def testFn(name: String,
```

Comment:
nit: since this specific to this Suite, it's clear what it's doing but maybe 'test/assertVariant` or something?